README.md

```diff
@@ -50,17 +50,19 @@ See https://matomo.org/docs/requirements/
 
 See https://matomo.org/docs/installation/
 
+(Alternatively you could also [deploy Matomo from the Git repository.](https://matomo.org/faq/how-to-install/faq_18271/))
```

Comment:
This we don't want to recommend anymore, only for development. We even mention it on the page itself.